transformer/classes/dockerfile/pythondockerfilegenerator.go

```diff
@@ -0,0 +1,248 @@
+/*
+ *  Copyright IBM Corporation 2021
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package dockerfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/konveyor/move2kube/common"
+	"github.com/konveyor/move2kube/environment"
+	"github.com/konveyor/move2kube/qaengine"
+	"github.com/konveyor/move2kube/types/qaengine/commonqa"
+	transformertypes "github.com/konveyor/move2kube/types/transformer"
+	"github.com/konveyor/move2kube/types/transformer/artifacts"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	pythonExt           = ".py"
+	pythonMain          = "main"
+	pythonManage        = "manage"
+	pythonService       = "python"
+	djangoService       = "django"
+	requirementsTxtFile = "requirements.txt"
+	//DetectedPortsConfigType contains the detected ports
+	DetectedPortsConfigType transformertypes.ConfigType = "DetectedPortsConfigType"
+	//RequirementsTxtConfigType points to the requirements.txt file if it's present
+	RequirementsTxtConfigType transformertypes.ConfigType = "RequirementsTxtConfigType"
+	// DjangoProjectConfigType is set to true for Django projects
+	DjangoProjectConfigType transformertypes.ConfigType = "DjangoProjectConfigType"
+	// MainPythonFilesPathType points to the .py file path which contains main function
+	MainPythonFilesPathType transformertypes.PathType = "MainPythonFilesPathType"
+)
+
+var (
+	pythonMainRegex       = regexp.MustCompile(`__main__`)
+	djangoDependencyRegex = regexp.MustCompile(`django`)
```

Comment:
Won't a contains command work in both these cases? do we need regex?

Reply:
Yes, contains command will work in both the cases. So, we don't need regex. 
Done.